.github/workflows/autolink-readthedocs-previews-to-prs.yaml

```
@@ -0,0 +1,21 @@
+name: Add readthedocs preview link to pull requests
+
+on:
+  pull_request:
```

```suggestion
  pull_request_target:
```
Maybe using `pull_request_target:` might help, @sgibson91?

> This event runs in the context of the base of the pull request, rather than in the context of the merge commit, as the pull_request event does. This prevents execution of unsafe code from the head of the pull request that could alter your repository or steal any secrets you use in your workflow. This event allows your workflow to do things like label or comment on pull requests from forks. Avoid using this event if you need to build or run code from the pull request.

---

This may be something that we have to merge first to test

---

I verified that this should work elsewhere (but I also verified the initial version of this workflow worked 🙄 )

- Workflow file: https://github.com/sgibson91/test-multilingual-sphinx/blob/main/.github/workflows/pull-request-links.yaml
- Test PR: https://github.com/sgibson91/test-multilingual-sphinx/pull/9

So maybe just merge and see? It can always be reverted if it doesn't work